src/front-end/Dockerfile

```diff
@@ -1,6 +1,10 @@
 FROM --platform=linux/amd64 docker.io/node:10-jessie
 COPY . /usr/app
 WORKDIR /usr/app
-EXPOSE 3000
+RUN npm install
+RUN NODE_ENV=production npm run front-end:build
```

Comment:
Does this production environment variable mess up local development? i.e. will local development always be prod-like

Reply:
Locally, its behaviour suggests it's a red herring. Modifying the input doesn't effect the output with what can be observed inside a running container using both directory size and file composition of the build/front-end directory. Various flavours of input: 

- `NODE_ENV=production`, 
- `NODE_ENV=${NODE_ENV}`
- `NODE_ENV=development` 
- omitting a node environment declaration altogether

produce the same build output: 
![image](https://user-images.githubusercontent.com/2048170/141182212-fc1cb497-0b47-4a71-b229-7185659b0988.png)